website/content/commands/login.mdx

```diff
@@ -56,5 +56,7 @@ or file.
 - `-from-kubernetes-service=<string>` - The name of the Kubernetes service to get the server address from when using the -from-kubernetes flag.
 - `-from-kubernetes-secret=<string>` - The name of the Kubernetes secret that has the Waypoint token when using the -from-kubernetes flag.
 - `-from-kubernetes-namespace=<string>` - The name of the Kubernetes namespace that has the Waypoint token when using the -from-kubernetes flag.
+- `-from-kubernetes-kubeconfig=<string>` - Path to the kubeconfig file for the cluster Waypoint was installed into when using the -from-kubernetes flag.
+- `-from-kubernetes-context=<string>` - The name of the Kubernetes context that Waypoint was installed into when using the -from-kubernetes flag.
```

Comment:
Considering Mitchell's comment, it may be worth mentioning here that if omitted we'll try to use the default env var values

Reply:
I would actually just specify the `EnvVar` setting and the help will auto-include this.

Reply:
Hmm, looks like we only have an EnvVar setting for plugin docs, and even that seems to not work: https://github.com/hashicorp/waypoint/issues/3135